src/main/utility/proc_maps.rs

```diff
@@ -82,7 +82,7 @@ pub struct Mapping {
     pub offset: usize,
     pub device_major: i32,
     pub device_minor: i32,
-    pub inode: i32,
+    pub inode: u64,
```

Comment:
Maybe worth adding adding a unit test that requires 64 bits, and/or a comment that we know 32 bits isn't enough.

Reply:
Added a unit test with a large inode value.